### DISCUSSION TITLE
In the absence of docs, can anyone share how ovretls works?

### DISCUSSION THREAD

**Discussion by harismuzaffer** [ShadowsocksR-Live/overtls#35] [Q&A]
I read the ReadMe but it is too brief. I need someone share some more details about the basics of overtls - specifically the Principle section of ReadMe and the core pieces involved in achieving this. Thanks

**Comment by ssrlive** [comment]
What are your difficulties?

**Comment by ssrlive** [comment]
1. Prepare your VPS and a domain name pointing to your VPS.
2. Install overtls with the [script](https://github.com/ShadowsocksR-Live/overtls/blob/master/install/overtls-install.sh).
3. Done.

Please write here while you meeting any trouble.
